exchange/howto/run-seednode.adoc

```diff
@@ -4,108 +4,306 @@
+- please be responsive when addressed
 
 == System requirements for hosting machine
 
-. Min. 2 GB of RAM
-. 10 GB disk space (SSD)
-. 2 TB network traffic
-. UPS (uninterruptible power supply)
-. Uptime of > 99.9%
+- Min. 4 GB of RAM
+- 400 GB of disk space
+- 2 TB network traffic
+- Uptime of > 99.9%
+- > 10000 file handles available (check `ulimit -Sn`)
+
+And please try to stay away from cheap hosting providers! They tend to not care about QOS, the hardware is slow, the occasionally even turned a service off without warning - ask us how we know...
+
+## Get a seed node up and running
+
+You can run your seed node in many ways. The guide you are currently reading applies to a (debian-based) unix system with systemd available. Furthermore, we assume that only one seed node is operated per host. Please be aware that the guide is only a guide and not a step by step copy and paste how to, as we ignore for example the need for the occational root access and file permissions, swap, general opsec, among other things. In case your host differs from our reference system, we are sure you can handle yourself.
+
+There are four pieces to the puzzle of successfully running a seed node:
+
+- a Bitcoin full node
+- the Bisq seednode itself and
+- system health reports
+
+and last but not least, the all-important firewall.
+
+=== Firewall
+
+Although every port that is to be used only by localhost clients should be hardened by configuration and source code already, we recommend whitelisting stuff that is actually needed. Here is an example on how to configure iptables accordingly (the config drops any incoming connections except connection from localhost, SSH and responses.):
+
+	iptables -P INPUT DROP && iptables -A INPUT -i lo -j ACCEPT && iptables -A INPUT -p tcp --dport 22 -j ACCEPT && iptables -A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+
+[NOTE]
+====
+Make sure you execute *all `iptables` commands at once*! Executing only the first one is going to kick you out of your ssh session and does not let you back in.
+====
+
+=== Bitcoin Node
+
+Start by setting up a user for the bitcoin core stuff.
+
+	useradd -r -m bitcoind
+	cd /home/bitcoind
+
+Download the Bitcoin Core binaries https://bitcoin.org/en/download[here] to this directory, unpack it and, for updating convenience, create a symlink to the necessary binaries (so you only have to change the symlink on update):
+
+	tar xzf bitcoin-0.18.0-x86_64-linux-gnu.tar.gz
+	ln -s bitcoin-0.18.0/bin/bitcoind bitcoind
+	ln -s bitcoin-0.18.0/bin/bitcoin-cli bitcoin-cli
+
+Create a systemd service file `bitcoind.service` in the systemd service path or your operating system (something like `/usr/lib/systemd/system/`) and adapt it to your needs. We recommend to create a user `bitcoind` for service hardening reasons (). In the end, it should look like
+
+----
+[Unit]
+Description=Bitcoind
+After=network.target
+
+[Service]
+ExecStart=bitcoind -daemon \
+                   -printtoconsole \
```

Comment:
Is this desired?

Reply:
what exactly do you mean?

Reply:
I am just not sure if it is necessary to printtoconsole for a daemon process.

Reply:
ah, well. The printtoconsole option sends outputs to stderr. On a system running systemd stderr ends up in the systems journal accessible via journalctl.